sql-statements/sql-statement-admin-checksum-table.md

```diff
@@ -6,13 +6,29 @@ category: reference
 
 # ADMIN CHECKSUM TABLE
 
-The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data and indexes of a table. This statement is used by programs such as TiDB Lightning to ensure that import operations have completed successfully.
+The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data and indexes of a table.
+
+<CustomContent platform="tidb">
+
+The [checksum](/tidb-lightning/tidb-lightning-glossary.md#checksum) is calculated over the table data and properties like the `table_id`. This means that two tables with the same data but a different `table_id` will get different checksums.
+
+This statement is used to calculate a checksum that can be compared with the checksum that [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) has calculated locally. This statement is usually executed by [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md). The same method is also used by the [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) statement.
```

Comment:
it seems that DM also executes this statement if `checksum-physical` is at its default value "required"

ref: https://docs.pingcap.com/tidb/dev/task-configuration-file-full#task-configuration-file-template-advanced